### PR TITLE
feat: add CreateAuthenticationExecutionFlow

### PR DESCRIPTION
**Pull request by weixiao-huang**
Thanks for your contribution!
Hi, if there is an issue, that your PR adresses, please link it!

### CODE REVIEW

**Comment by Nerzal** [comment]
Hi, could you please resolve the merge conflicts?

**Comment by weixiao-huang** [comment]
All conflicts have been resolved

**Comment by Nerzal** [comment]
Hi again :) 

The code looks good so far. 
Could you please also add a test for that function? :)

**Comment by weixiao-huang** [comment]
I added tests but I cannot test it in my local machine, so please approve the test ci, thanks

File: client_test.go [hunk 1]
```diff
@@ -6229,4 +6258,30 @@ func TestGocloak_CreateAuthenticationFlowsAndCreateAuthenticationExecution(t *te
...
+			)
+			require.NoError(t, err, "Failed to delete authentication execution")
+			execDeleted = true
+		}
+		if execDeleted && execFlowFound {
+			break
+		}
+	}
+	require.True(t, execDeleted, "Failed to delete authentication execution, no execution was deleted")
+	require.True(t, execFlowFound, "Failed to find authentication execution flow")
```
Comment (bastianccm) [inline client_test.go:6286]: This should go on block up, before deleting the flow, as the flow removes the execution too. (Simply cut the lines here and move them up to line 6244). Then the test will pass.

**Review by bastianccm — CHANGES_REQUESTED** [review]

**Review by bastianccm — APPROVED** [review]

**Comment by weixiao-huang** [comment]
Hello, could you please approve the ci to test?

**Comment by bastianccm** [comment]
Hi @Nerzal can you take a look into this change? Thank you!

**Comment by Nerzal** [comment]
LGTM thank you for the contribution! 
I'll create a new Release this week, which includes all PRs i merged during the last days

**Pull request merged**